common/io_debug.c

```diff
@@ -1,17 +0,0 @@
-#include <ccan/err/err.h>
```

Comment:
This was removed but the `lightningd/Makefile` `channeld/Makefile` still seems to use it?

Reply:
Thanks!  Fixed and rebased...